GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationAddressController.cs

```diff
@@ -202,9 +203,13 @@ private IActionResult CheckChangesAction(ChangeOrganisationAddressViewModel view
 
         private void SaveChangesAndAuditAction(ChangeOrganisationAddressViewModel viewModel, Organisation organisation)
         {
-            string oldAddressString = organisation.GetLatestAddress().GetAddressString();
+            OrganisationAddress latestAddress = organisation.GetLatestAddress();
+            string oldAddressString = latestAddress?.GetAddressString();
 
-            RetireOldAddress(organisation);
+            if (latestAddress != null)
+            {
+                RetireOldAddress(organisation);
+            }

```

Comment:
Not a big deal, but was just wondering if we could just do:
```
string oldAddressString = organisation.GetLatestAddress()?.GetAddressString()

if (oldAddressString != null)
{
RetireOldAddress(organisation)
}
```

That way we don't need the `latestAddress` variable. I could be missing something, but just though I'd check

Reply:
Nice! 👍